contracts/storage/ParametersStorage.sol

```diff
@@ -3,8 +3,13 @@
 pragma solidity ^0.8.4;
 
 import {Hub} from "../Hub.sol";
+import {Named} from "../interface/Named.sol";
+import {Versioned} from "../interface/Versioned.sol";
+
+contract ParametersStorage is Named, Versioned {
```

Comment:
Will we break any apps/nodes by changing parameters storage?

Reply:
No, functions are the same. I just changed the way of storing, because we reached the limit on number of variables